.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,111 @@
+name: "Bug report: Netdata"
+description: "Submit a report and help us improve our free and open-source Netdata Agent"
+title: "[Bug]: "
+labels: ["bug", "needs triage"]
+body:
+  - type: markdown
+    attributes:
+      value: "### Thank you for contributing to our project!"
+  - type: markdown
+    attributes:
+      value: |
+        <img src="https://img.shields.io/github/v/release/netdata/netdata.svg?label=latest%20stable"> <img src="https://img.shields.io/badge/dynamic/xml?url=https://storage.googleapis.com/netdata-nightlies/latest-version.txt&label=latest%20nightly&query=/text()">
```

Comment:
Should maybe the links point to docs for helping update? And at the same time have here a badge with current nightly and stable versions ?

Reply:
What docs?

Reply:
Maybe this: https://learn.netdata.cloud/docs/agent/packaging/installer/update ?